rootfs-builder/alpine/config.sh

```diff
@@ -22,3 +22,5 @@ INIT_PROCESS=kata-agent
 # List of zero or more architectures to exclude from build,
 # as reported by  `uname -m`
 ARCH_EXCLUDE_LIST=()
+
+[ "$SECCOMP" = "yes" ] && PACKAGES+=" libseccomp" || true
```

Comment:
hi @nitkon, the `|| true` part is not needed, if you check above your discussion with @jodh-intel

Reply:
One more thing: could you add the same in `suse/config.sh` (libseccomp2) and in `template/config_template.sh` (by default commented out, and you could include a brief stating your intentions)?